babel.config.json

```diff
@@ -1,3 +1,7 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-typescript"]
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-typescript",
+    "@babel/preset-react"
```

Comment:
We can remove the React preset as we don't need to support JSX in this Hooks library.

Reply:
Removed